res/css/views/settings/_AvatarSetting.scss

```diff
@@ -41,8 +83,9 @@ limitations under the License.
     & > img,
     .mx_AvatarSetting_avatarPlaceholder {
         display: block;
-        height: $font-88px;
-        border-radius: 4px;
+        height: 90px;
+        border-radius: 90px;
```

Comment:
the radius of the border exceeds half the length of an edge, 50% is clearer

Reply:
The convention has always been to use the length of any side when using a square.